oioioi/contests/notifications.py

```diff
@@ -16,27 +16,41 @@ def notification_function_submission_judged(arguments):
     request.timestamp = datetime.now().replace(tzinfo=utc)
 
     # Check if the final report is visible to the user
-    if not pi.contest.controller.can_see_submission_score(
+    if pi.contest != None and not pi.contest.controller.can_see_submission_score(
```

Comment:
I think we should call pi.controlller.*, as then we will also ask the problemcontroller if there is no contest.

Reply:
Yeah, you're right